modules/nw-route-specific-annotations.adoc

```diff
@@ -29,6 +29,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist` | Sets a whitelist for the route. |
 |`haproxy.router.openshift.io/hsts_header` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`haproxy.router.openshift.io/log-send-hostname` | Sets the hostname field in the Syslog header. Uses the hostname of the system.
```

Comment:
In addition, we can also mention that  "this gets applied/enabled by default if any ingress API logging method (sidecar or Syslog facility) is enabled for the router"

Other than that, lgtm.

Reply:
Thank you! Added.